packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.2.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/V3.1.1...flutter-3.2.0) (2022-06-22)
```

Comment:
The latest dart tag is `dart-3.1.1`. Since this PR only contains bug fixes, the next dart release will be `dart-3.1.2`.

Reply:
I explained about this in the previous PR.

As we have dependencies update, some changing the MAJOR, I always change the MINOR, as it can break applications due to conflicting versions.

Version 3.2 is correct.